package/MDAnalysis/analysis/rdf.py

```diff
@@ -77,28 +90,86 @@
    :members:
    :inherited-members:
 
+   .. attribute:: bins
+
+      :class:`numpy.ndarray` of the centers of the `nbins` histogram
+      bins.
+
+   .. attribute:: edges
+
+      :class:`numpy.ndarray` of the `nbins + 1` edges of the histogram
+      bins.
+
+   .. attribute:: rdf
+
+      :class:`numpy.ndarray` of the :ref:`radial distribution
+      function<equation-gab>` values for the :attr:`bins`.
+
+   .. attribute:: count
+
+      :class:`numpy.ndarray` representing the radial histogram, i.e.,
+      the raw counts, for all :attr:`bins`.
+
 
 Site-specific radial distribution function
 ------------------------------------------
 
 :class:`InterRDF_s` calculates site-specific radial distribution
 functions. Instead of two groups of atoms it takes as input a list of pairs of
-AtomGroup, ``[[A, B], [C, D], ...]``. Give the same ``A`` and ``B`` to
-:class:`InterRDF_s`, the output will be a list of RDFs between ``A1`` and
-``B1``, ``A1`` and ``B2``, ``A2`` and ``B1``, ``A2`` and ``B2`` (and similarly
-for ``C`` and ``D``). These site-specific radial distribution functions are
-typically calculated if one is interested in the solvation shells of a ligand
-in a binding site or the solvation of specific residues in a protein. A common
-use case is to choose ``A`` and ``C`` to be AtomGroups that only contain a
-single atom and ``W`` all solvent molecules: ``InterRDF_s(u, [[A, W], [B,
-W]])`` will then produce the RDF of solvent around atom ``A[0]`` and around
-atom ``B[0]``.
+AtomGroup, ``[[A, B], [C, D], ...]``. Given the same ``A`` and ``B`` to
+:class:`InterRDF_s`, the output will be a list of individual RDFs between
+``A1`` and ``B1``, ``A1`` and ``B2``, ``A2`` and ``B1``, ``A2`` and ``B2`` (and
+similarly for ``C`` and ``D``). These site-specific radial distribution
+functions are typically calculated if one is interested in the solvation shells
+of a ligand in a binding site or the solvation of specific residues in a
+protein. A common use case is to choose ``A`` and ``C`` to be AtomGroups that
+only contain a single atom and ``W`` all solvent molecules: ``InterRDF_s(u,
+[[A, W], [B, W]])`` will then produce the RDF of solvent around atom ``A[0]``
+and around atom ``B[0]``.
```

Comment:
@VOD555 I think that this sentence is not really correct: if we choose `W` to be all solvent molecules then we will get one RDF for _each solvent molecule_ – right? So the user still need to average over `rdf[i][0, :]`. The memory requirements might be very high for including hundreds of thousands of solvent molecules so I am not sure if we should recommend it here.

Reply:
Yes, it will return one RDF for each solvent molecule. Maybe we can have another option to control whether to average over `rdf` for each pairs when doing the calculation, but for this version, I think we should not recommend it here.

Reply:
Ok, I remove the sentence.